docs/src/components/DemoPage.tsx

```diff
@@ -1,27 +1,27 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import { Panel, FormControl, Grid, Row, Col } from 'react-bootstrap';
+import store from '../state/store';
 import HTML5Backend from 'react-dnd-html5-backend';
 import { DragDropContext } from 'react-dnd';
 import {
     IField,
     IFieldContext,
     IFormError,
-    FieldOptionEditor,
-    FieldSelector,
-    FormBuilder,
     FormInput,
     FormDisplay,
 } from 'react-dynamic-formbuilder';
 
+import FormBuilder from './FormBuilder';
```

Comment:
The consumer must create a Higher Order Component (HOC) from the original FormBuilder and pass in store reference. This means creating separate component file for their HOC